scripts/alltools.tcl

```diff
@@ -437,3 +445,17 @@ proc ordnumber {string} {
   }
   return $string
 }
+
+proc ucwords {string} {
+	list asent
+	foreach word [split [join $string]] {
```

Comment:
This has to be just `[split $string]` to iterate over the words, otherwise it breaks on special characters. `[split [join $string]]` does nothing because split and join do the exact opposite of each other